snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/components/SnomedConceptApiTest.java

```diff
@@ -945,4 +927,51 @@ public void testPendingMoveUpdateOnConceptWithSetDescriptions() {
 		});
 	}
 	
+	@Test
+	public void testConceptInactivationModuleChanges() {
+		final String conceptId = createNewConcept(branchPath);
+		String sourceRelationshipId = createNewRelationship(branchPath, conceptId, Concepts.HAS_DOSE_FORM, Concepts.MODULE_SCT_MODEL_COMPONENT);
+		String destinationRelationshipId = createNewRelationship(branchPath, Concepts.MODULE_SCT_MODEL_COMPONENT, Concepts.HAS_DOSE_FORM, conceptId);
+		CodeSystemURI codeSystemURI = CodeSystemURI.branch(SnomedTerminologyComponentConstants.SNOMED_SHORT_NAME, branchPath.getPath());
```

Comment:
`branchPath` is absolute, so this test looks for a branch path named `MAIN/MAIN/...`, and so can not find what it was looking for.

Reply:
Done